src/rdkafka.h

```diff
@@ -3130,6 +3131,18 @@ int rd_kafka_event_log (rd_kafka_event_t *rkev,
 
 
 /**
+ * @brief Extract stats from the event.
+ *
+ * Event types:
+ *  - RD_KAFKA_EVENT_STATS
+ *
+ * @returns stats json string.
+ */
+RD_EXPORT
+char* rd_kafka_event_stats (rd_kafka_event_t *rkev);
```

Comment:
I suggest making this a const, and also add `@remark` about the lifetime of the string, e.g.:
`@remark the returned string will be freed automatically along with the event object`